src/projections/labrd.cpp

```diff
@@ -6,7 +6,7 @@
 #include "proj.h"
 #include "proj_internal.h"
 
-PROJ_HEAD(labrd, "Laborde") "\n\tCyl, Sph\n\tSpecial for Madagascar";
+PROJ_HEAD(labrd, "Laborde") "\n\tCyl, Sph\n\tlat_0= Special for Madagascar";
```

Comment:
Perhaps ?
```suggestion
PROJ_HEAD(labrd, "Laborde") "\n\tCyl, Sph\n\tSpecial for Madagascar\n\tlat_0=";
```

Reply:
I'm not sure what the expected format is here...
All projections I found had 2 lines max where  the first line provided the projection info and the second line arguments and sometimes a bit of text appended to it. I don't know what happens to other users of the headers if we add a third line.

I'm fine with either form. But perhaps we should add a comment on the expected format while we are at it.